data/KrishnaTheCoder756.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Krishna Sarathi Ghosh",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Python, Web Developer.",
+  "avatar": "https://pbs.twimg.com/profile_images/1614971735903993856/FC2ibH_8_400x400.png"
```

Comment:
Please add a missing comma `,` here to validate your json!! 🛠

```suggestion
  "avatar": "https://pbs.twimg.com/profile_images/1614971735903993856/FC2ibH_8_400x400.png",
```

**Tip:** you can commit the requested changes directly using the <kbd>Commit suggestions</kbd> button above! 🙃